waspc/cli/Main.hs

```diff
@@ -17,7 +17,7 @@ import           Command.CreateNewProject (createNewProject)
 import           Command.Db               (runDbCommand, studio)
 import           Command.Db.Migrate       (migrateSave, migrateUp)
 import           Command.Start            (start)
-import qualified Command.Telemetry        as Telemetry
+import           Command.Telemetry        (considerSendingData, telemetry)
```

Comment:
I would rather we keep it qualified, otherwise when reading `considerSendingData` in the code, it is not very obviuos what it is about.

Reply:
Updated! :)